### DISCUSSION TITLE
is there away to search for particular currencies?

### DISCUSSION THREAD

**Discussion by The1Waterman** [firefly-iii/firefly-iii#9394] [General]
is there away to search for particular currencies, I've tried "currency_code, currency_name and currency_symbol"  with no luck, ie currency_symbol:€ , currency_name:euro  currency_code:eur

**Comment by JC5** [comment]
https://docs.firefly-iii.org/references/firefly-iii/search/

**Comment by The1Waterman** [comment]
is there away to list the search results by description rather than the default date?
